### PR TITLE
indexer: sequencer/rollup `+on-watch` -> ``this`

### PR DESCRIPTION
**Pull request by hosted-fornet**

### CODE REVIEW

File: app/sequencer.hoon [hunk 1]
```diff
@@ -58,17 +58,14 @@
 ++  on-watch
   |=  =path
   ^-  (quip card _this)
-  ?.  =(%available status.state)
-    ~|("%sequencer: error: got watch while not active" !!)
   ::  open: anyone can watch
+  :: ?.  =(%available status.state)
+  ::   ~|("%sequencer: error: got watch while not active" !!)
```
Comment (hosted-fornet) [inline app/sequencer.hoon:63]: Please note this check is now commented to avoid breaking fakenet setup.

**Comment by dr-frmr** [comment]
Reviewing and deploying on prerelease testnet to make sure we don't break anything

File: app/sequencer.hoon (hunk 1 above)
Comment (dr-frmr) [reply app/sequencer.hoon:63]: I think we can safely delete this whole comment block

Comment (hosted-fornet) [reply app/sequencer.hoon]: 0acbc0cc removes comment block

**Review by dr-frmr — APPROVED** [review]

**Pull request merged**